File: dev/RecommenderAttacksAnalytics/src/main/java/com/codetrix/util/Logger.java
package com.codetrix.util;

public class Logger {
	
	public static void log(String str){
		
		System.out.println(str);
	}
	
	public static void logWarning(String str){
		
		System.err.println("Warning: " + str);
	}
	
	public static void logError(String str){
		
		System.err.println("Error: " + str);
	}

}

File: main/RecommenderSystemAnalytics/RecommenderAttacksAnalytics/Utililty/Logger.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace RecommenderAttacksAnalytics.Utility
{
    class Logger
    {

        public static void log(string str)
        {
            Console.Out.WriteLine(str);
        }

        public static void logWarning(string str)
        {
            
        }

        public static void logError(string str)
        {
            Console.Out.WriteLine("ERROR: " + str);
        }

    }
}
